Tools/Postprocessing/C_Src/WritePlotfileToASCII.cpp

```diff
@@ -20,7 +20,7 @@ PrintUsage (const char* progName)
             << "coordinates, row/column formatting, etc." << std::endl << std::endl;
     
     Print() << "Usage:" << '\n';
-    Print() << progName << " infile=inputFileName" << '\n' << '\n';
+    Print() << progName << " infile=inputFileName (optional comp_in_line = 1)" << '\n' << '\n';
```

Comment:
```suggestion
    Print() << progName << " infile=inputFileName [comp_in_line = 1]" << '\n' << '\n';
```
Square brackets are commonly used to indicate optional parameters. https://en.wikipedia.org/wiki/Usage_message